man/ola_recorder.1

```diff
@@ -32,6 +34,11 @@ Print
 version information
 .IP "--duration <uint32_t>"
 The length of time (seconds) to run for.
+.IP "--start <uint_32_t>"
+Time (milliseconds) in show file to start playback from.
+.IP "--stop <uint32_t>"
+Time (milliseconds) in show file to stop playback at.  If the show file is
+shorter, this option is ignored.
```

Comment:
It would be nice to be able to do exactly a 30s loop (from 5 to 35), even if your showfile is a bit short (say 34, but then just warn that it's run out of showfile).

Reply:
I'll double check; this should already be possible.  Just missed updating docs.